src/index.html

```diff
@@ -14,8 +14,45 @@
     >
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet"> 
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img 
+      class="card__image" 
+      src="./images/imac.jpeg" 
+      alt="Apple iMac" 
+      >	
+
+      <h1 class="card__description">	
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)	
+      </h1>	
+      
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="card__review">
+        <div class="card__stars stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__count">Reviews: 5</span>
+      </div>
+      
+      <div class="card__price">
+        <p class="card__price-label">Price:</p>
+        <p class="card__price-tag">$2,199</p>
+      </div>
+
+      <a
+        href="/"
+        class="card__button"
+        data-qa="hover">
+        Buy
+      </a>
```

Comment:
close at the same indent
```suggestion
      <a
        href="/"
        class="card__button"
        data-qa="hover"
      >
        Buy
      </a>
```